REQUIRE

```diff
@@ -1,5 +1,8 @@
-julia 0.6
+julia 0.7
 MathProgBase 0.6 0.8
 ReverseDiffSparse 0.8 0.9
 ForwardDiff 0.5 0.8
 Calculus
+LinearAlgebra
```

Comment:
I don't think these are needed, they're baked into Julia.

Reply:
I think you're right

Reply:
You need to do `using Compat.LinearAlgebra` for it to work on both v0.6 and v0.7